test_unstructured/metrics/test_text_extraction.py

```diff
@@ -339,6 +339,21 @@ def test_prepare_string(text, expected):
     assert text_extraction.prepare_str(text, standardize_whitespaces=True) == expected
     assert text_extraction.prepare_str(text) == text
 
+@pytest.mark.parametrize(
+    ("input_text", "expected_output"),
+    [
+        ('"Hello"', '"Hello"'),  # Basic double quotes
+        ("'Hello'", "'Hello'"),  # Basic single quotes
+        ('„Hello"', '"Hello"'),  # German-style quotes to standard
+        ('"Hello"', '"Hello"'),  # Fancy double quotes to standard
```

Comment:
is this test covering all possible types of quote marks? I found quite long list for unicode: https://hexdocs.pm/ex_unicode/Unicode.Category.QuoteMarks.html